index.js

```diff
@@ -71,6 +71,7 @@ module.exports = {
         "no-labels": 2,
         "no-lone-blocks": 2,
         "no-loop-func": 2,
+        "no-multiple-empty-lines": [2, { max: 1 }],
```

Comment:
Hmm `maxEOF` needs to be 1 to avoid clashes with `"eol-last": 2,` no? Didn't test it but I'd assume it would raise an error on one of the rules if `maxEOF` is set to 0.